google/auth/jwt.py

```diff
@@ -381,9 +382,8 @@ def with_claims(self, issuer=None, subject=None, audience=None,
                 claim will be used.
             subject (str): The `sub` claim. If unspecified the current subject
                 claim will be used.
-            audience (str): the `aud` claim. If not specified, a new
-                JWT will be generated for every request and will use
-                the request URI as the audience.
+            audience (str): the `aud` claim. If unspecified the current
```

Comment:
I prefer to reserve code for well.. code. The claim names are part of the JWT spec, so I prefer to just em them.